.github/workflows/ci.yml

```diff
@@ -60,7 +60,8 @@ jobs:
               compiler: gcc-12,    cxxstd: '03,11,14,17,20', os: ubuntu-22.04 }
           - { name: Collect coverage, coverage: yes,
               compiler: gcc-8,     cxxstd: '03,11',          os: ubuntu-20.04, install: 'g++-8-multilib', address-model: '32,64' }
-
+          - { name: Coverage lcov v2.0, coverage: yes,
+              compiler: gcc-8,     cxxstd: '03,11',          os: ubuntu-20.04, install: 'g++-8-multilib libcapture-tiny-perl libdatetime-perl', address-model: '32,64', lcov-version: 'v2.0' }
```

Comment:
```suggestion
              compiler: gcc-8,     cxxstd: '03,11',          os: ubuntu-20.04, install: 'g++-8-multilib libcapture-tiny-perl', address-model: '32,64', lcov-version: 'v2.0' }
```

Wasn't that 2nd package only required for genhtml which we don't use?

Reply:
About the packages: I verified by omitting all packages, got an error, fixed the error with one package, got another error, etc.

Reply:
Ok, just wanted to double check that this isn't a debug left-over or so.